index.html

```diff
@@ -8,27 +8,50 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-        <header id="header">
+        <header class="header header_desktop">
             <img class ="image_geometry_1" src="images/image_geometry_1.png"/>
            
             <nav><img class ="ic_menu" src="images/ic_menu.png"/></nav>
 
         </header>
+       
+        <ul class="menu">
+            <li class="links"><a href="#hello">Hello</a></li>
+            <li class="links"><a href="#portfolio">Portfolio</a></li>
+            <li class="links"><a href="#about">About</a></li>
+            <li class="links"><a href="#contact">Contact</a></li>
+         </ul>
+         
+      
+       
 
-        <section id="headline">
+        <section class="headline">
+            <div class="mb-headline">
             <h1 class="profileName">Harshita Jain</h1>
             
             <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me. </p>
             <button class="startbtn">Start collaboration</button>
+            <img class="dotImg2" src="images/image_dots.svg">
+        </div>
+        
+        <div>
+            <img class="geometry_big" src="images/image-geometry_11.svg"/>
+        </div>
         </section>
         <img class="geometry" src="images/image_geometry_2.png"/>
 
-        <section id="works">
+        <section class="works">
            
-            <h2 class="pro">Projects</h2>
-       <div class="grid-items">
-        <div class="project">
-            <img class="pr-img1" src="images/Rectangle 21.png">
+         <h2 class="pro">Projects</h2>
+
+         <div class="dots">
+            <img class="dotImg" src="images/image_dots.svg">
+         </div>
+         
+
+
+         <div class="project pr-dsk1">
+            <img class="pr-img1" src="images/Rectangle 21.svg">
             <div class="project-ins">
                 <h3>Project name goes here</h3>
                 <ul>
```

Comment:
- [ ] Kindly add the project's content as shown in Figma. Right now, I can only see the picture 😓 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200124411-d814733c-5e96-4113-adfa-2091685103b3.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200124429-b4f81856-e312-4256-8b59-1aa87db7b1bf.png)

Reply:
Fixed.